iped-app/src/main/java/iped/app/home/processmanager/ProcessManager.java

```diff
@@ -0,0 +1,228 @@
+                commandArgs.add(currentEvidence.getAditionalComands());
+            }
+        }
+        return commandArgs;
+    }
+
+    public ArrayList<String> getCaseOutputCommand(Path caseOutput){
+        ArrayList<String> commandArgs = new ArrayList<>();
+        commandArgs.add("-o");
+        commandArgs.add( "\"" + caseOutput.toString() + "\"" );
+        return commandArgs;
+    }
+
+    public ArrayList<String> getProfileCommand(String profileName){
+        ArrayList<String> commandArgs = new ArrayList<>();
+        if( StringUtils.isEmpty(profileName) )
+            return commandArgs;
+        commandArgs.add("-profile");
+        commandArgs.add( profileName );
+        return commandArgs;
+    }
+
+    public ArrayList<String> getIpedSearchAppJarCommand(Path caseOutput){
+        ArrayList<String> cmds = new ArrayList<>();
+        cmds.add("-jar");
+        cmds.add(Paths.get(caseOutput.toString(), "iped", "lib", "iped-search-app.jar").toString() );
+        return cmds;
+    }
+
+    public ArrayList<String> getIpedJarCommand(){
+        ArrayList<String> cmds = new ArrayList<>();
+        cmds.add("-jar");
+        cmds.add(Paths.get(System.getProperty(IConfigurationDirectory.IPED_APP_ROOT), "iped.jar").toString() );
+        return cmds;
+    }
+
+    public String getJarBinCommand(){
+        String javaBin = "java";
+        File embeddedJRE = Paths.get(getRootPath(), "jre").toFile();
+        File javaHome = new File(System.getProperty("java.home"));
+        if (!javaHome.equals(embeddedJRE)) {
+            String warn = Util.getJavaVersionWarn();
+            if (warn != null) {
+                System.err.println(warn);
+            }
+        }
+        if (org.apache.tika.utils.SystemUtils.IS_OS_WINDOWS) {
+            javaBin = Paths.get(javaHome.getPath(), "bin", "java.exe").toString();
+        }
+        return javaBin;
+    }
+
+    private String getRootPath(){
+        String rootPath = null;
+        try {
+            URL url = ProcessManager.class.getProtectionDomain().getCodeSource().getLocation();
+            if ("true".equals(System.getProperty("Debugging"))) {
+                rootPath = System.getProperty("user.dir");
+            } else {
+                rootPath = new File(url.toURI()).getParent();
+                // test for report generation from case folder
+                if (rootPath.endsWith("iped" + File.separator + "lib")) { //$NON-NLS-1$ //$NON-NLS-2$
+                    rootPath = new File(url.toURI()).getParentFile().getParent();
+                }
+            }
+        }catch (Exception e){
+            e.printStackTrace();
+        }
+        return rootPath;
+    }
+
+    public void openMulticase(File multiCaseFile){
+        ArrayList<String> commandList =  new ArrayList<>();
+        commandList.add(getJarBinCommand());
+        commandList.addAll(getIpedSearchAppJarCommand(CasePathManager.getInstance().getCasePath().toPath()));
```

Comment:
I think this will make it independent from the current path from where the `MainFrame.java` is executed.

but will this config UI only be executed from inside an existing case, or will there be an external .exe?

Reply:
Hi @FelipeFcosta 


> Hi @thiagofuer, maybe you could pass a `casePath` to this method `openMulticase` and use it here just like in the `openSingleCase(Path casePath)`?

I dont think so, because the multiCaseFile arg is a referent to the text file containing all cases path to be opened


> so in line 122:
> `commandList.addAll(getIpedSearchAppJarCommand(casePath));`

If i change CasePathManager to multicase i will lose the txt file reference
To open a single case i only need to call the ipedsearch, on multicases i need all cases references

@patrickdalla i created the CasePathManager class to set all cases references, but i see that you have change it on the MainFrame class, so if the best way to get the case reference is getin it from Configuration class, so some refactoring will be necessary. 

Can you confirm please?

Reply:
> I dont think so, because the multiCaseFile arg is a referent to the text file containing all cases path to be opened
> 
> > so in line 122:
> > `commandList.addAll(getIpedSearchAppJarCommand(casePath));`
> 
> If i change CasePathManager to multicase i will lose the txt file reference To open a single case i only need to call the ipedsearch, on multicases i need all cases references
> 

the text file reference will still be there, you just add the casePath arg, like I said: 
`pm.openMulticase(caseList.get(0), multiCaseFile)`

what I'm saying is that there's no need for the call:
`CasePathManager.getInstance().getCasePath().toPath()`
you can use one of the cases path as a starting point for the multicase and it will work the same, but now it's not dependent on the current case, this is important if this UI config should be able to be executed from outside any iped case.

Reply:
"but i see that you have change it on the MainFrame class"
Sorry, I pushed some local debugging environment adaptations erroneously, so I reverted it.
But when reverting, I do not know how can I config my debug env. I tryed to config the variable testPath in CasePathManager but it wasn't enough. Maybe you have some clue.

Reply:
It seems that, when debugging, the "target" path where the compiled version is deployed does not contain the "iped" sub folder.

Reply:
I have replaced all occurrences of "iped" in CasePathManager string to the constant that gets the value from IIPEDSource.MODULE_DIR (already defined as "iped"). 

Now it works in my debug environent if I only change this constant to an empty string and the testPath variable.

So please review if it loads in a normal environment.

Thanks.

Reply:
> "but i see that you have change it on the MainFrame class" Sorry, I pushed some local debugging environment adaptations erroneously, so I reverted it. But when reverting, I do not know how can I config my debug env. I tryed to config the variable testPath in CasePathManager but it wasn't enough. Maybe you have some clue.

To use the debug mode i do the folow:
1-Run Maven compile and install
2-go to the release folder and rename iped
/home/user/workspace/iped/target/release/iped4.x-snapshot to /home/user/workspace/iped/target/release/iped 
3-Go to CasePathManager and Set the testpath var value pointing to release folder
`private final File testPath = new File("/home/user/workspace/iped/target/release"); `

To compile without tests errors i skip the maven tests (there are some certificate tests that throws errors)